### PR TITLE
OSDOCS-5987-networking: batch of bugs for 4.14 RNs

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):

Issue:
https://issues.redhat.com/browse/OSDOCS-5987

Link to docs preview:
https://66433--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-bug-fixes:~:text=OCPBUGS%2D10714)-,Previously%2C%20when,-configuring%20DNS%20for

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -1210,6 +1210,23 @@ With this update, when creating a `DNSRecord` custom resource (CR) for a gateway
 
 * Previously, `oc explain route.spec.tls.insecureEdgeTerminationPolicy` documented the incorrect possible options that could be confusing some some users. With this release, the API documentation has been updated so that it shows the correct possible options for `insecureEdgeTerminationPolicy`. This is an API documentation fix only. (link:https://issues.redhat.com/browse/OCPBUGS-11393[*OCPBUGS-11393*])
 
+* Previously, CoreDNS terminated unexpectedly if a user created an `EndpointSlice` port without a port number. With this update, validation was added to CoreDNS to prevent it from unexpectedly terminating. (link:https://issues.redhat.com/browse/OCPBUGS-19805[*OCPBUGS-19805*])
+
+* Previously, the OpenShift Router directed traffic to a route with a weight of `0 ` when it had only one back end. With this update, the router will not send traffic to routes with a single backend with weight `0`. (link:https://issues.redhat.com/browse/OCPBUGS-16623[*OCPBUGS-16623*])
```
Comment (mburke5678) [inline release_notes/ocp-4-14-release-notes.adoc:1215]: _back end_ or _backend_? 

Is OpenShift Router_ a thing? I see only one reference in an Atom search (in MS docs)

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -1210,6 +1210,23 @@ With this update, when creating a `DNSRecord` custom resource (CR) for a gateway
...
+
+Previously, the Ingress Operator did not check status of DNS records in public hosted zones when updating the Operator status.
+This caused the Ingress Operator to report the DNS status as `ready` when there could be errors in DNS records in public hosted zones. Now, the Ingress Operator checks the status of both public and private hosted zones fixing the issue. (link:https://issues.redhat.com/browse/OCPBUGS-15978[*OCPBUGS-15978*])
```
Comment (mburke5678) [inline release_notes/ocp-4-14-release-notes.adoc:1220]:
```suggestion
This caused the Ingress Operator to report the DNS status as `ready` when there could be errors in DNS records in public hosted zones. Now, the Ingress Operator checks the status of both public and private hosted zones, which fixes the issue. (link:https://issues.redhat.com/browse/OCPBUGS-15978[*OCPBUGS-15978*])
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -1210,6 +1210,23 @@ With this update, when creating a `DNSRecord` custom resource (CR) for a gateway
...
+
+* Previously, the CoreDNS bufsize setting was configured as 512 bytes. Now, the maximum size of the buffer for {product-title} CoreDNS is 1232 bytes. This modification enhances DNS performance by reducing the occurrence of DNS truncations and retries. (link:https://issues.redhat.com/browse/OCPBUGS-15605[*OCPBUGS-15605*])
```
Comment (mburke5678) [inline release_notes/ocp-4-14-release-notes.adoc:1222]: Does `bufsize` need mark-up?

File: release_notes/ocp-4-14-release-notes.adoc [hunk 4]
```diff
@@ -1210,6 +1210,23 @@ With this update, when creating a `DNSRecord` custom resource (CR) for a gateway
...
+
+* Previously, the Ingress Operator would specify the `spec.template.spec.hostNetwork: true` on a router deployment without specifying the `spec.template.spec.containers[*].ports[*].hostPort`. This caused the API server to set a default value for each port's `hostPort` filed, which the Ingress Operator would then detect as an external update and attempt to revert it. Now, the Ingress Operator no longer incorrectly performs these updates resolving the issue. (link:https://issues.redhat.com/browse/OCPBUGS-14995[*OCPBUGS-14995*])
```
Comment (mburke5678) [inline release_notes/ocp-4-14-release-notes.adoc:1224]:
```suggestion
* Previously, the Ingress Operator would specify the `spec.template.spec.hostNetwork: true` parameter on a router deployment without specifying the `spec.template.spec.containers[*].ports[*].hostPort`. This caused the API server to set a default value for each port's `hostPort` field, which the Ingress Operator would then detect as an external update and attempt to revert it. Now, the Ingress Operator no longer incorrectly performs these updates resolving the issue. (link:https://issues.redhat.com/browse/OCPBUGS-14995[*OCPBUGS-14995*])
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 5]
```diff
@@ -1210,6 +1210,23 @@ With this update, when creating a `DNSRecord` custom resource (CR) for a gateway
...
+
+* Previously, the DNS Operator logged the error message on startup `cluster-dns-operator startup has an error message: [controller-runtime] log.SetLogger(...) was never called, logs will not be displayed:` misleading users. Now, the error message is not displayed on startup. (link:https://issues.redhat.com/browse/OCPBUGS-14395[*OCPBUGS-14395*])
```
Comment (mburke5678) [inline release_notes/ocp-4-14-release-notes.adoc:1226]:
```suggestion
* Previously, the DNS Operator logged the `cluster-dns-operator startup has an error message: [controller-runtime] log.SetLogger(...) was never called, logs will not be displayed:` error message on startup, which could mislead users. Now, the error message is not displayed on startup. (link:https://issues.redhat.com/browse/OCPBUGS-14395[*OCPBUGS-14395*])
```

Comment (mburke5678) [reply release_notes/ocp-4-14-release-notes.adoc:1226]: Would this be moreaccurate?
> Previously, the DNS Operator incorrectly logged the...

File: release_notes/ocp-4-14-release-notes.adoc [hunk 6]
```diff
@@ -1210,6 +1210,23 @@ With this update, when creating a `DNSRecord` custom resource (CR) for a gateway
...
+
+* Previously, the Ingress Operator was leaving the "spec.internalTrafficPolicy", "spec.ipFamilies", and "spec.ipFamilyPolicy" fields unspecified for `NodePort` and `ClusterIP` type services. The API would then set default values for these fields, which the Ingress Operator would try to revert. With this update, the Ingress Operator specifies an initial value and fixes the error caused by API default values. (link:https://issues.redhat.com/browse/OCPBUGS-13190[*OCPBUGS-13190*])
```
Comment (mburke5678) [inline release_notes/ocp-4-14-release-notes.adoc:1228]:
```suggestion
* Previously, the Ingress Operator was leaving the "spec.internalTrafficPolicy", "spec.ipFamilies", and "spec.ipFamilyPolicy" fields unspecified for `NodePort` and `ClusterIP` services. The API would then set default values for these fields, which the Ingress Operator would try to revert. With this update, the Ingress Operator specifies an initial value and fixes the error caused by the API default values. (link:https://issues.redhat.com/browse/OCPBUGS-13190[*OCPBUGS-13190*])
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 7]
```diff
@@ -1900,6 +1917,12 @@ For more information about early access, xref:../updating/understanding_updates/
 
 * Currently, users cannot modify the `interface-specific` safe sysctl list by updating the `cni-sysctl-allowlist` config map in the `openshift-multus` namespace. As a workaround, you can modify, either manually or with a Daemon Set, the file `/etc/cni/tuning/allowlist.conf` on the node or nodes. (link:https://issues.redhat.com/browse/OCPBUGS-11046[*OCPBUGS-11046*])
 
+* In {product-title} 4.14, all nodes use Linux control group version 2 (cgroup v2) for internal resource management in alignment with the default RHEL 9 configuration. However, if you apply a performance profile in your cluster, the low latency tuning features associated with the performance profile do not support cgroup v2.
++
+As a result, if you apply a performance profile, all nodes in the cluster will reboot to switch back to the cgroup v1 configuration. This reboot includes control plane nodes and worker nodes that were not targeted by the performance profile.
```
Comment (mburke5678) [inline release_notes/ocp-4-14-release-notes.adoc:1922]:
```suggestion
As a result, if you apply a performance profile, all nodes in the cluster reboot to switch back to the cgroup v1 configuration. This reboot includes control plane nodes and worker nodes that were not targeted by the performance profile.
```

**Comment by mburke5678** [comment]
@JoeAldinger A few nits. Feel free to ignore. Otherwise LGTM

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (JoeAldinger) [reply release_notes/ocp-4-14-release-notes.adoc:1215]: ISG says back end is preferred in most cases. https://www.ibm.com/docs/en/ibm-style?topic=word-usage#b

**Pull request closed without merging**